src/global/Firebase/FirebaseContext.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import Firebase from './Firebase';
+
+const FirebaseContext = React.createContext<Firebase | null>(null);
```

Comment:
Maybe should we use this helper to create context - it prevents empty initialization.
https://github.com/typescript-cheatsheets/react-typescript-cheatsheet/blob/master/README.md#context

Reply:
Thanks, I will use it.